.ci/openshift_e2e.sh

```diff
@@ -18,12 +18,37 @@ set -u
 # print each command before executing it
 set -x
 
-trap 'Catch_Finish $?' EXIT SIGINT
+function bumpPodsInfo() {
+    NS=$1
+    TARGET_FLD=${ARTIFACTS_DIR}/$NS-info
```

Comment:
It's just a quick fix, if you have better folders/files structure proposal, let me know.